internal/plugin/server_test.go

```diff
@@ -132,6 +132,7 @@ func TestCDIAllocateResponse(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
```suggestion
	for i := range testCases {
	tc := testCases[i]
```

Reply:
Sure, I can make this change. However, the change I've made is a well known and widely followed practice. We have done the same in the other CNT repos

Reply:
As a note. This is a false positive if go1.22 is used.

Reply:
@ArangoGutierrez I've made the change